src/client/core/transport.js

```diff
@@ -1,81 +1,25 @@
 import * as hammerheadAPI from './deps/hammerhead';
```

Comment:
Here and in all other places: why it's called hammerheadAPI why not just hammerhead?